README.md

```diff
@@ -1,2 +1,21 @@
-# news-for-maintainers
-Announcements of changes impacting collection contributors and maintainers
+# News for Maintainers
+Announcements of changes impacting Ansible collection contributors and maintainers.
+
```

Comment:
```suggestion

[Current open issues](https://github.com/ansible-collections/news-for-maintainers/issues)
```

Reply:
I would add another newline after this though.

Reply:
I'll add the newline, thanks!